airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -185,7 +185,7 @@ class Config:
 
 class Data(BaseModel):
     class Config:
-        extra = Extra.forbid
+        extra = Extra.allow
```

Comment:
Isn't allowing extra risky? Forbidding extra is a nice way to have consistency guarantees across metadata files